source/_patterns/basics/forms/input.hbs

```diff
@@ -3,3 +3,72 @@
   <input type="text" class="form-control" id="inlineFormInput" placeholder="Jane Doe">
   <button type="submit" class="btn btn-primary">Submit</button>
 </form>
+
+<form>
```

Comment:
I don't see this update on the site: http://blog.andresgalante.com/patternfly-atomic/

Am I missing something?